### PR TITLE
1. fix gesture listener signature argument type

### PR DESCRIPTION
**Pull request by MinuteReversal**
1. fix gesture listener signature argument type.
2. remove the google_font dependency from example

### CODE REVIEW

**Review by MinuteReversal — COMMENTED** [review]
I want fix this lib can work in flutter 3 project

**Review by MinuteReversal — COMMENTED** [review]
I'm test the change in web runing mode. It work well

**Review by MinuteReversal — COMMENTED** [review]
remove unused import

**Review by MinuteReversal — COMMENTED** [review]
support MathML

File: lib/src/widgets/selection/gesture_detector_builder.dart [hunk 1]
```diff
@@ -145,30 +145,32 @@ class MathSelectionGestureDetectorBuilder {
...
   @protected
-  void onDragSelectionEnd(DragEndDetails details) {
+  void onDragSelectionEnd(TapDragEndDetails details) {
     /* Subclass should override this method if needed. */
   }
 
   @protected
-  void onDragSelectionUpdate(
-      DragStartDetails startDetails, DragUpdateDetails updateDetails) {
+  void onDragSelectionUpdate(TapDragUpdateDetails updateDetails) {
```
Comment (Decodetalkers) [inline lib/src/widgets/selection/gesture_detector_builder.dart:173]: here is no startDetail now, you have miss to edit it, line 175

Comment (MinuteReversal) [reply lib/src/widgets/selection/gesture_detector_builder.dart:173]: I saved startDetail to the startDetails variable on line 156.

Comment (MinuteReversal) [reply lib/src/widgets/selection/gesture_detector_builder.dart:173]: I saved startDetail to the startDetails variable on line 156.

**Comment by Sese-Schneider** [comment]
@MinuteReversal the fix for Flutter 3.10 has been published.
Please open an additional PR for the MathML support!

**Pull request merged**